tests/search/test_main.py

```diff
@@ -100,6 +100,56 @@ def test_entity_types(
     entity_types.get_by_text("Activity").click()
     expect(page.get_by_text("Showing 1 of 1 items")).to_be_visible()
     page.screenshot(
-        path="tests_search_test_main-test_index-on-select-entity-1-found.png"
+        path="tests_search_test_main-test_entity_types-on-select-entity-1-found.png"
     )
     entity_types.get_by_text("Activity").click()
+
+
+@pytest.mark.integration
+@pytest.mark.usefixtures("load_dummy_data")
+def test_load_search_params(
+    frontend_url: str,
+    writer_user_page: Page,
+) -> None:
+    page = writer_user_page
+    page.goto(
+        f"{frontend_url}/?q=help&page=1&entityType=ContactPoint&entityType=Consent"
+    )
+
+    # check 1 item is showing
+    expect(page.get_by_text("Showing 1 of 1 items")).to_be_visible()
+    page.screenshot(
+        path="tests_search_test_main-test_load_search_params-on-params-loaded.png"
+    )
+    search_result_cards = page.locator(".search-result-card")
+    expect(search_result_cards).to_have_count(1)
+    expect(search_result_cards).to_contain_text("help@contact-point.two")
+
+    # check entity types are loaded from url
+    entity_types = page.get_by_test_id("entity-types")
+    unchecked = entity_types.get_by_role("checkbox", checked=False)
+    expect(unchecked).to_have_count(11)
+    checked = entity_types.get_by_role("checkbox", checked=True)
+    expect(checked).to_have_count(2)
+
+
+@pytest.mark.integration
+@pytest.mark.usefixtures("load_dummy_data")
+def test_push_search_params(
```

Comment:
I would like to see pushing the query string tested, too.

Reply:
fixed https://github.com/robert-koch-institut/mex-editor/blob/9962f3c2d30846cb866f989c6e9fd865dc29cddd/tests/search/test_main.py#L160-L166

Reply:
whoa, cool highlight!